content/post/frontend_talks_2017.md

```diff
@@ -0,0 +1,142 @@
+
+First, she starts telling us about animations through the recent history until the times of Macromedia Flash and the appearance of the open standards we use today for animations on the web: HTML5, JS and CSS3.
+
+The first option for web animations is using the HTML5 `canvas` element to draw and animate. She also explains how to do that with ember components.
+This has a few advantages like not needing DOM operations, having a good performance and being a powerful way for the developer to animate things, but it's too complicated for creators and hard for accessibility.
+
+Then, as an alternative, we can use the WebAnimations API, which is still not supported in all browsers (but we can use with a polyfill). It's very similar to CSS3 keyframes, so for people already familiar with that it will be an easy experience. As with the `canvas` HTML5 element, she explains as well how to put all this into an Ember component.
+Even though it does not have a performance as good as HTML5 canvas, the WebAnimations API makes it possible to build accessible animations and are easier for the creator.
+
+Video is still pending to be uploaded.
+
+### Communication is the key to a healthy relationship: the love story of Elm and JavaScript, _Miguel Molina_
+
+Just before lunch, we had the last talk, a lightning talk about how to communicate between Elm and JavaScript by [Miguel Molina](https://twitter.com/erizocosmico) (yours truly), fullstack developer at source{d}.
+
+In this talk, several methods to communicate between Elm and JavaScript are explored after a brief introduction of what is Elm and why Elm and not other compile-to-JS languages.
+
+3 methods of interoperability are discussed:
+* Native modules, which are hard to write, inconvienient for the package writer and not documented on purpose.
+* Ports, which are the recommended way of communicating elm and JavaScript using unidirectional inbound and outbound channels to pass messages.
+* Flags, which is a way to pass some initialization values or configuration from JavaScript to Elm.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/-2KQOywMnXM" frameborder="0" allowfullscreen></iframe>
+
+## Afternoon session
+
+### Decorating JavaScript, _Sergio Arbeo_
+
+After we filled our bellies with nice food, we had the first long talk of the afternoon, explaining what are decorators and what are their uses in JavaScript, by [Sergio Arbeo](https://twitter.com/serabe), lead platform engineer at source{d}.
+
+In this talk, Sergio explains that decorators have been worked on since March, 2015 by Yehuda Katz and they are (and have been) in stage-2 for 11 months, so the spec is probably not going to suffer many more changes until it gets to stage 3 and can be implemented by browsers.
+
+A decorator is a new syntax to declaratively modify the shape of class declarations, either the class itself or its members using functions.
```

Comment:
"declaratively modify the shape of class declarations"

How about ~declaratively~... or is this intentionally? ;-)